pkg/event/event.go

```diff
@@ -84,6 +84,7 @@ func NewEventQueue(c chan Events, flushThreshold int, flushInterval time.Duratio
 	eq := &EventQueue{
 		C:              c,
 		flushThreshold: flushThreshold,
+		flushInterval:  flushInterval,
```

Comment:
oh, ha, that's a bug you fixed there, but could you please submit this as a separate PR?